<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMLServerV3Class</class>
 <widget class="QMainWindow" name="XMLServerV3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMLServerV3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidgetTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>141</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>TableName</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetClient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>561</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Client-ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Client-IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OpHandle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Access time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Queued</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>XML Table data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetCommand">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>561</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Access Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>XML-ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OpeHandle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Command</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Milisec</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Command</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWriteOut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>WriteOut</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxShowAll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>76</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show all</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>561</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSetting</sender>
   <signal>clicked()</signal>
   <receiver>XMLServerV3Class</receiver>
   <slot>slotSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonWriteOut</sender>
   <signal>clicked()</signal>
   <receiver>XMLServerV3Class</receiver>
   <slot>slotWriteOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSetting()</slot>
  <slot>slotWriteOut()</slot>
 </slots>
</ui>
